src/main/java/frc/robot/commands/IntakingNoteCommand.java

```diff
@@ -0,0 +1,44 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.ParallelRaceGroup;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.ConveyorSubsystem;
+import frc.robot.subsystems.ElevatorSubsystem;
+import frc.robot.subsystems.FloorIntakeSubsystem;
+import frc.robot.subsystems.FlyWheelSubsystem;
+import frc.robot.subsystems.PivotSubsystem;
+import frc.robot.commands.ConveyorCommands.ConveyLineBreak;
+import frc.robot.commands.FloorIntakeCommands.FI_LinebreakCommand;
+import frc.robot.commands.ShootingPosCommands.BasePosition;
+
+public class IntakingNoteCommand extends SequentialCommandGroup{
+    FloorIntakeSubsystem floorIntake;
+    ConveyorSubsystem conveyor;
+    ElevatorSubsystem elevator;
+    PivotSubsystem pivot;
+    FlyWheelSubsystem flyWheel;
+
+    public IntakingNoteCommand(FloorIntakeSubsystem floorIntake, ConveyorSubsystem conveyor,
+        ElevatorSubsystem elevator, PivotSubsystem pivot, FlyWheelSubsystem flywheel){
+        
+        this.floorIntake = floorIntake;
+        this.conveyor = conveyor;
+        this.elevator = elevator;
+        this.pivot = pivot;
+        this.flyWheel = flywheel;
+
+        addCommands(
+            new ParallelCommandGroup(
```

Comment:
What is your intent for these commands here? Is the idea that:
- `FI_LinebreakCommand` and `BasePosition` run in parallel
- Then `ConveyLineBreak` runs?

Or do you want all of them to run in parallel?

Right now these commands will all run in sequence because you are chaining them together with `andThen`.

If you want them all to run in sequence, the code would look more like:

```
addCommands(
  new FI_LinebreakCommand(floorIntake),
  new BasePosition(pivot, elevator),
  new ConveyLineBreak(conveyor, flywheel)
)
```

If you wanted to run the first 2 in parallel, it would look more like:

```
addCommands(
  new ParallelCommandGroup(
    new FI_LinebreakCommand(floorIntake),
    new BasePosition(pivot, elevator),
  ),
  new ConveyLineBreak(conveyor, flywheel)
)
```


One more note is that we should remember that `BasePosition` (and all the other positional commands) finish immediately (even though the motors will keep running after the command ends due to the PID controller). You probably want to update these commands to check that the encoders have reached the configured setpoint to compute `isFinished` instead of always returning `true`

Reply:
I made the base position be the first thing that runs, should I just run everything in sequence after that? I can also do a parallel command where the floor intake spins with the conveyor after the Floor Intake LB is tripped once. 

So,
Base Pos,
Floor Intake LB, (wait to see data) 
Conveyor LB + Floor Intaking Command (run in parallel) 
Then finish the command

Reply:
I think based on the discord discussion, you probably want to run:
1. Base Position (wait for this to finish)
2. Floor Intake + Conveyor LB in parallel

If the pivot/elevator is in base position, you don't need to deal with the floor intake linebreak sensor since that's only needed to prevent the handoff of the note from the floor intake to the cage

Reply:
got it, sounds good